The ``4-print_square`` module
=============================


Using ``print_square``
----------------------
>>> print_square = __import__('4-print_square').print_square

>>> print_square(4)
####
####
####
####
>>> print_square(10)
##########
##########
##########
##########
##########
##########
##########
##########
##########
##########
>>> print_square(0)
>>> print_square(1)
#
>>> print_square(-1)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/kusimo/alx-higher_level_programming/0x07-python-test_driven_development/4-print_square.py", line 9, in print_square
    raise ValueError("size must be >= 0")
ValueError: size must be >= 0
>>> print_square(0.1)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/kusimo/alx-higher_level_programming/0x07-python-test_driven_development/4-print_square.py", line 7, in print_square
    raise TypeError("size must be an integer")
TypeError: size must be an integer
>>> print_square("five")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/kusimo/alx-higher_level_programming/0x07-python-test_driven_development/4-print_square.py", line 7, in print_square
    raise TypeError("size must be an integer")
TypeError: size must be an integer
>>> print_square(1.2)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/kusimo/alx-higher_level_programming/0x07-python-test_driven_development/4-print_square.py", line 7, in print_square
    raise TypeError("size must be an integer")
TypeError: size must be an integer
>>> print_square(None)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/kusimo/alx-higher_level_programming/0x07-python-test_driven_development/4-print_square.py", line 7, in print_square
    raise TypeError("size must be an integer")
TypeError: size must be an integer
>>> print_square(-3)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/kusimo/alx-higher_level_programming/0x07-python-test_driven_development/4-print_square.py", line 9, in print_square
    raise ValueError("size must be >= 0")
ValueError: size must be >= 0
>>> print_square(a)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
NameError: name 'a' is not defined
>>> print_square(15, 9)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: print_square() takes 1 positional argument but 2 were given

